app/webpack/stylesheets/components/_phase_banner.scss

```diff
@@ -0,0 +1,33 @@
+$laa-uat-colour: govuk-colour("orange");
```

Comment:
Can we add one in for local.  Perhaps this:
```
$app-local-colour: govuk-colour("bright-purple");
```

Reply:
👍 have made suggested change.